widgets/query-resources-by-category-widget.php

```diff
@@ -1,258 +1,293 @@
-		</p>
-
-		<p>
-			<label for="<?php echo $this->get_field_id('type');?>"><?php _e('Dataset type:');?></label>
-			<input class="widefat" id="<?php echo $this->get_field_id('type');?>" name="<?php echo $this->get_field_name('type');?>" type="text" value="<?php echo esc_attr($type);?>" placeholder="<?php _e('dataset, library_record, etc..');?>">
-		</p>
-
-		<?php $limit = !empty($instance['limit']) ? $instance['limit'] : -1 ?>
-		<p>
-			<label for="<?php echo $this->get_field_id( 'limit' ); ?>"><?php _e( 'Select max number of posts to list (-1 to show all):' ); ?></label>
-			<input class="widefat" id="<?php echo $this->get_field_id('limit');?>" name="<?php echo $this->get_field_name('limit');?>" type="number" value="<?php echo $limit;?>">
-		</p>
-		<p>
-            <label for="<?php echo $this->get_field_id('more_text'); ?>"><?php _e('More dataset: Link label', 'wpckan'); ?></label>
+                echo $output;
+
+                if (odm_language_manager()->get_current_language() == 'en') {
+                    if (!empty($instance['more_link']) && $instance['more_link'] != "") :
+                        echo '<div style="text-align:right"><a href="' . $instance['more_link'] . '" target="_blank">' . $instance['more_text'] . '</a></div>';
+                    endif;
+                } else {
+                    if (!empty($instance['more_link_local']) && $instance['more_link_local'] != "") :
+                        echo '<div style="text-align:right"><a href="' . $instance['more_link_local'] . '" target="_blank">' . $instance['more_text_local'] . '</a></div>';
+                    endif;
+                }
+
+                echo $args['after_widget'];
+            endif;
+        endif;
+    }
+
+    /**
+     * Outputs the options form on admin
+     *
+     * @param array $instance The widget options
+     */
+    public function form($instance)
+    {
+        $title              = !empty($instance['title']) ? __(sanitize_text_field($instance['title']), 'wpckan') : __('Related resources by category', 'wpckan');
+        $type               = !empty($instance['type']) ? $instance['type'] : 'dataset';
+        $search_field       = !empty($instance['search_field']) ? sanitize_text_field($instance['search_field']) : 'taxonomy';
+        $organization       = isset($instance['organization']) ? $instance['organization'] : -1;
+        $organization_list  = [];
+        $sort               = isset($instance['sort']) ? $instance['sort'] : 'metadata_modified+desc';
+        $template           = isset($instance['template']) ? $instance['template'] : 'dataset-list';
+        $more_text          = !empty($instance['more_text']) ? $instance['more_text'] : 'More...';
+        $more_text_local    = !empty($instance['more_text_local']) ? $instance['more_text_local'] : 'បន្ថែម...';
```

Comment:
Do you want to use `_e()` for this default text?  It looks like that's our localization method for all the rest of this plugin.

Reply:
Sorry that I missed seeing your suggestion. I will check and update the code. Thank you.